src/client/legacy.rs

```diff
@@ -1347,6 +1348,22 @@ impl Builder {
         self
     }
 
+    /// Provide a timer to execute background HTTP2 tasks
+    ///
+    /// See the documentation of [`h2::client::Builder::timer`] for more
+    /// details.
+    ///
+    /// [`h2::client::Builder::timer`]: https://docs.rs/h2/client/struct.Builder.html#method.timer
+    pub fn timer<M>(&mut self, timer: M) -> &mut Self
+    where
+        M: Timer + Send + Sync + 'static,
+    {
+        #[cfg(feature = "http2")]
+        self.h2_builder.timer(timer);
```

Comment:
This does nothing without `http2` feature, but I think that is ok since it will once we add it to the pool(?)

Reply:
Yes, that sounds right.